[PATCH] add fallback function for apply_simd_restrictions() on  non-x86/ppc
--- a/libaften/cpu_caps.h	2023-01-08 15:33:03.281595530 -0500
+++ b/libaften/cpu_caps.h	2023-01-08 15:33:19.244895966 -0500
@@ -26,6 +26,7 @@
 #include "ppc_cpu_caps.h"
 #else
 static inline void cpu_caps_detect(void){}
+static inline void apply_simd_restrictions(AftenSimdInstructions *simd_instructions){}
 #endif
 
 #endif /* CPU_CAPS_H */
